code/web/sys/SystemVariables.php

```diff
@@ -272,12 +274,36 @@ static function getObjectStructure($context = ''): array {
 				'label' => 'App Scheme',
 				'description' => 'Scheme used for creating deep links into the app',
 			],
-			'trackIpAddresses' => [
-				'property' => 'trackIpAddresses',
-				'type' => 'checkbox',
-				'label' => 'Track IP Addresses',
-				'description' => 'Determine if IP Addresses should be tracked for each page view',
-				'default' => false,
+			'dataProtectionRegulations' => [
+				'property' => 'dataProtectionRegulations',
+				'type' => 'section',
+				'label' => 'Data Protection Regulations',
+				'hideInLists' => true,
+				'expandByDefault' => true,
+				'properties' => [
+					'cookieStorageConsent' => [
+						'property' => 'cookieStorageConsent',
+						'type' => 'checkbox',
+						'label' => 'Require Cookie Storage Consent',
+						'description' => 'Require users to consent to cookie storage before using the catalog',
+						'default' => false,
+					],
+					'cookiePolicyHTML' => [
+						'property' => 'cookiePolicyHTML',
+						'type' => 'html',
+						'label' => 'Cookie Policy',
+						'description' => 'HTML of cookie policy to display to users',
+						'default' => 'This body has not yet set a cookie storage policy, please check back later.',
+						'hideInLists' => true,
+					],
+					'trackIpAddresses' => [
```

Comment:
This has a slight code smell.. I like the move to nest this inside an overall 'dataProtectionsRegulations' block.. but I don't see any change of referencing above to compensate for that move.  

Now I've done a quick cod review, I intend to live test it a bit ;P

Reply:
OK, confirmed nothing needs to change here.. I can see the checkbox happily in the page.